contentcuration/contentcuration/viewsets/file.py

```diff
@@ -61,8 +65,26 @@ def update(self, instance, validated_data):
                 ResourceSizeCache.reset_modified_for_file(instance)
 
         results = super(FileSerializer, self).update(instance, validated_data)
+        results.on_update()  # Make sure contentnode.content_id is unique
+
+        if results.contentnode:
+            results.contentnode.refresh_from_db()
+            if not len(results.contentnode.mark_complete()):
```

Comment:
Should we be handling this any differently? Basically `mark_complete` returns `List<errors>` when it is incomplete, is there anything I should do here w/ the information if there are issues?